Assets/Scripts/Items/GameItem.cs

```diff
@@ -7,11 +7,33 @@
 /// Parent class for both inventory and world items.
 /// </summary>
 public abstract class GameItem : Hoverable {
+    /*
+     *Property for whether this GameItem is inanimate or not
+     */
+    [SerializeField][HideInInspector]
+    private bool isAnimate;
+
+    //check if this game item is animate
+    public bool checkAnimate()
```

Comment:
You may as well make this a property instead of two separate functions:
public bool IsAnimate {
get { return isAnimate; }
private set { isAnimate = value; }
}
(We probably should not be able to set it from elsewhere.)